### PR TITLE
Update ViewPortHandler.swift

### PR DESCRIPTION
**Pull request by no33jou**
fix y precision .

### CODE REVIEW

File: Source/Charts/Utils/ViewPortHandler.swift [hunk 1]
```diff
@@ -434,7 +434,7 @@ open class ViewPortHandler: NSObject
     
     @objc open func isInBoundsTop(_ y: CGFloat) -> Bool
     {
-        return _contentRect.origin.y <= y
+        return _contentRect.origin.y <= y + 1.0
```
Comment (jjatie) [inline Source/Charts/Utils/ViewPortHandler.swift]: Why?

Comment (no33jou) [reply Source/Charts/Utils/ViewPortHandler.swift]: when iOS9 ，_contentRect.origin.y is 30.0，y is 2.999999986，so The y value error is plus or minus 1

Comment (liuxuan30) [reply Source/Charts/Utils/ViewPortHandler.swift]: I don't get it. what's the problem again?

**Comment by liuxuan30** [comment]
no update since then. closing.

**Pull request closed without merging**